makefile

```diff
@@ -423,31 +423,10 @@ docker_buildandpackage: docker_ok
 # Test targets        #
 #---------------------#
 
-ifndef NORACETEST
-ifeq (,$(findstring history_size,$(GORACE)))
-export GORACE:=$(GORACE) history_size=7
-endif
-ifeq (,$(findstring halt_on_error,$(GORACE)))
-export GORACE:=$(GORACE) halt_on_error=1
-endif
-ifeq (,$(findstring -race,$(GOTEST_FLAGS)))
-export GOTEST_FLAGS:=$(GOTEST_FLAGS) -race
-endif
-endif
```

Comment:
I suggest keeping these variables, and changing run-test.sh to something like
```
go test ${GOTEST_FLAGS} -p 1 ./...
```
The idea is two-fold:
1. The make user can run `make NORACETEST=1` to execute the tests without race checks
2. The make user can run `make GO_TEST_FLAGS="my custom flags"` to pass additional flags to go test.